### DISCUSSION TITLE
Using Gulp with Older Understrap-Child Projects?

### DISCUSSION THREAD

**Discussion by mikeoberdick** [understrap/understrap-child#339] [Q&A]
I've been using Understrap Child for quite some time.  I have about fifty local sites that were built using the gulp task runner and since the migration to the new Understrap Child theme those older sites no longer work with "gulp watch".  I tried dropping down the Node version to see if that was the issue but it's still not working.  Any ideas besides building my own gulp file?  I'm thinking it should still work but maybe it has something to do with updating to the latest Understrap parent theme?

**Comment by bacoords** [comment]
Hi @mikeoberdick  - 

The latest version of the child theme doesn't use the Gulp build process. Updating the parent theme shouldn't effect your child theme environment at all. If you're updating your child theme build process, take a look at the docs for the new build tools:
https://docs.understrap.com/#/understrap-child/npm

Also, you can always keep your current package.json and not change your build process if you don't want to.

**Comment by alexatlevel** [comment]
@mikeoberdick This is a super late reply but I have the same issue. I still develop new sites on the npm (not gulp) version but I have a large number of sites that I still maintain using the old gulp / bs4 framework. I've found that running nvm use 10 then checking for node -v 10.24.1 and npm -v 6.14.12 is a good setup for running gulp to compile sass and js one the older installs.
